src/scripts/website.sh

```diff
@@ -11,6 +11,13 @@ WEBSITE_SRC_DIR=./www-src
 rm -rf $WEBSITE_SRC_DIR $WEBSITE_DIR
 mkdir -p $WEBSITE_SRC_DIR
 
+# build manual as pdf for download
+sphinx-build -t website -c "$SPHINX_CONFIG" -b "latex" doc/manual latex
+cd latex
+pdflatex botan.tex
```

Comment:
The generated TOC is just fine, I just double-checked. As of now, we have no index of functions in the LaTex document. I would love to have one, as we have in the online manual, but I couldn't find any docs on how to do this. If you have a solution, that would be great.

Anyway, the CI build fails here, and I figured out why. sphinx generates a warning in case the botan.pdf file is not found. I explicitly put an `only:: html and website` directive around the download role in `index.rst`, but sphinx seems to ignore it when processing the download role. From reading https://github.com/sphinx-doc/sphinx/issues/2150 it seems the `only::` directive is only applied after processing the rEST, causing the warning regardless of whether the tags are set or not. The only way out I see is `cp`ing the PDF file in `website.sh` and hardcoding a link to https://botan.randombit.net/manual/botan.pdf into `index.rst`.